src/main/resources/application-dd.properties

```diff
@@ -1,5 +1,4 @@
 google.allowReuseExistingBuckets=true
-google.allowReuseExistingProjects=true
```

Comment:
Can this file be deleted?

Reply:
There's one of these for Dan and Rori. I think since they are both still at the Broad, it's acceptable to keep these around.